src/applications/registry.json

```diff
@@ -861,9 +861,9 @@
     }
   },
   {
-    "appName": "28-1900 Vocational Rehabilitation",
-    "entryName": "28-1900-chapter-31",
-    "rootUrl": "/vre/28-1900",
+    "appName": "28-8832-planning-and-guidance",
+    "entryName": "28-8832-planning-and-career-guidance",
+    "rootUrl": "/vre/28-8832",
     "template": {
```

Comment:
Just realized, this replaces the current registry entry for chapter 31, which we still need. Instead, a new entry for chapter 36 should be added to the registry, probably just below this one.

Reply:
the code for the existing chapter 31 is not in production nor will it ever be. What we are building is taking the place of that code.